### PR TITLE
Advanced mesh hiding

### PR DESCRIPTION
**Pull request by Jaimi**
feature merge

### CODE REVIEW

**Comment by kenamis** [comment]
there's still some submesh support that needs to be added.  I'll try to get to it soon.

**Comment by virror** [comment]
@Jaimi @kenamis Is this ready for review?

**Comment by Jaimi** [comment]
Yeah - all my changes are checked in.

File: UMAProject/Assets/UMA/Core/StandardAssets/UMA/Scripts/Editor/MeshHideInspector.cs [hunk 1]
```diff
@@ -0,0 +1,394 @@
...
+            serializedObject.ApplyModifiedProperties();
+
+            if (beginSceneEditing)
+            {
+                // This has to happen outside the inspector
+                EditorApplication.delayCall += CreateSceneEditObject;
+            }
+        }
+
+        private void CloseWindow()
```
Comment (virror) [inline UMAProject/Assets/UMA/Core/StandardAssets/UMA/Scripts/Editor/MeshHideInspector.cs]: Maybe remove this one?

File: UMAProject/Assets/UMA/Core/StandardAssets/UMA/Scripts/Editor/MeshHideInspector.cs [hunk 2]
```diff
@@ -0,0 +1,394 @@
...
+                }
+                _meshData.submeshes[i] = new SubMeshTriangles();
+                _meshData.submeshes[i].triangles = new int[newTriangles.Count];
+                newTriangles.CopyTo(_meshData.submeshes[i].triangles);
+            }
+        }
+
+        private void CreateSceneEditObject()
+        {
+
```
Comment (virror) [inline UMAProject/Assets/UMA/Core/StandardAssets/UMA/Scripts/Editor/MeshHideInspector.cs]: Lots of space ; )

**Pull request merged**